unity/Hello_World/Assets/MixedRealityToolkit.Generated/CustomProfiles/New MixedRealityToolkitConfigurationProfile.asset

```diff
@@ -31,7 +31,7 @@ MonoBehaviour:
   xrsdkBoundarySystemType:
     reference: Microsoft.MixedReality.Toolkit.XRSDK.XRSDKBoundarySystem, Microsoft.MixedReality.Toolkit.Providers.XRSDK
   boundaryVisualizationProfile: {fileID: 11400000, guid: 6d28cce596b44bd3897ca86f8b24e076, type: 2}
-  enableTeleportSystem: 1
+  enableTeleportSystem: 0
```

Comment:
What do these options relate to?

Reply:
https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/teleport-system/teleport-system?view=mrtkunity-2021-05

I don't think it ever actually did anything since we're on the HoloLens but I figured it couldn't hurt to disable it.